BaseCode/App/Globals/AppConstants.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  AppConstants.swift
+//  BaseCode
+//
+//  Created by BrainX IOS Dev on 25/08/2023.
+//
+
+import Foundation
+
+struct AppConstants {
+    
+    static let empty = ""
+}
```

Comment:
Constants are useful for storing values that are helpful for business logic. empty is not a constant. If you still want to use empty store in extension of string